test/e2e/apps/disruption.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
+	policyv1 "k8s.io/api/policy/v1"
 	policyv1beta1 "k8s.io/api/policy/v1beta1"
```

Comment:
So I think the test failures in #84740 is primarily because the feature flag wasn't updated, so the disruption controller isn't running in those tests. I have updated this PR to address that and will also update #84740.
Another issue here is that some of the pdb e2e tests use the eviction API, which of course is not GA. So I will look at the tests being promoted to conformance in #84740 and make sure those doesn't rely on Eviction. But this means we still need to import `policyv1beta1 "k8s.io/api/policy/v1beta1"` here.

Reply:
> Another issue here is that some of the pdb e2e tests use the eviction API, which of course is not GA. So I will look at the tests being promoted to conformance in #84740 and make sure those doesn't rely on Eviction. But this means we still need to import `policyv1beta1 "k8s.io/api/policy/v1beta1"` here.

ah, that makes sense for the import. I think we'll want to continue exercising the subresource in the test, there's little use for the PDB API otherwise